src/driver/main.cpp

```diff
@@ -811,7 +811,11 @@ struct main_command
 using namespace migraphx::driver; // NOLINT
 int main(int argc, const char* argv[])
 {
-    std::vector<std::string> args(argv + 1, argv + argc);
+    std::vector<std::string> cmd_args(argv, argv + argc);
+    std::string cmd_string = migraphx::to_string_range(cmd_args, " ");
+    std::cout << "Running " << cmd_string << std::endl;
```

Comment:
```suggestion
    std::cout << "Running: " << cmd_string << std::endl;
```

Reply:
Or it could be `[Running]: migraphx-driver perf resnet50.onnx` 

Add prepend `migraphx-driver` to it.

Reply:
I think the [ ] would be saved for a future PR to be the version of the code

Reply:
@umangyadav it will point to whatever the first command line argument the user invocated the driver from. Just to make it easier to debug.
@causten I added the version number in brackets now.